client/src/components/Registration/Registration.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from "react";
+import { Form, Button, Row, Col } from "react-bootstrap";
+const axios = require("axios");
+
+function Resisration() {
+  //state of first name
+  const [firstName, setFirstName] = useState("");
+  //state of last name
+  const [lastName, setLastName] = useState("");
+  //state of telephone number
+  const [telephoneNumber, setTelephonenumber] = useState("");
+  //state of email
+  const [email, setEmail] = useState("");
+
+  //get first name by onchange
+  function getFirstName(e) {
+    setFirstName(e.target.value);
+  }
+  console.log(firstName);
+  //get last name by onchange
+  function getLastName(e) {
+    setLastName(e.target.value);
+  }
+  console.log(lastName);
+  //get telephone number by onchange
+  function getTelephoneNumber(e) {
+    setTelephonenumber(e.target.value);
+  }
+  console.log(telephoneNumber);
+  //get email by onchange
+  function getEmail(e) {
+    setEmail(e.target.value);
+  }
+  console.log(email);
+  //submit data by cliking the button
+  //now, just console log
+  async function submitData() {
+    console.log(firstName);
+    console.log(lastName);
+    console.log(telephoneNumber);
+    console.log(email);
+    await axios.post("http://localhost:3000/someapi", {
+      first_name: firstName,
+      last_name: lastName,
+      telephone_Number: telephoneNumber,
+      e_mail: email,
+    });
+  }
+
+  return (
```

Comment:
Also, please add some comments to your code so we can all understand what is happening :)

Reply:
Thx! I added comment :)